docs/earthly-command/earthly-command.md

```diff
@@ -866,187 +1194,423 @@ Skips setting up the BuildKit container during bootstrapping. If needed, it will
+If the provider argument is given the CI application will automatically begin an OAuth flow with the given provider.
+If you are logged into the CLI the url will contain a token used to link your OAuth credentials to your Earthly user.
+
+#### Options
+
+##### `--provider`
+
+The provider to use when logging into the web ui.
+
+#### Examples
+
+##### Login to the CI application with GitHub
+
 * ```
-  earthly <command> --help
+  earthly web --provider=github
   ```
 
+## earthly docker-build
+
+#### Synopsis
+
+* ```
+  earthly [options] docker-build [--dockerfile <dockerfile-path>] [--tag=<image-tag>] [--target=<target-name>] [--platform <platform1[,platform2,...]>] <build-context-dir> [--arg1=arg-value]
+  ``` 
+
 #### Description
 
-Prints help information about earthly.
+The command `earthly docker-build` builds a docker image from a Dockerfile instead of an Earthfile.  
+The `<build-context-dir>` is the path where the Dockerfile build context exists. By default, it is assumed that a file named Dockerfile exists in that directory.
 
-## earthly --version
+Just like a regular build, `docker-build` can be used with a satellite. For example:
+```shell
+earthly docker-build --sat my-satellite --tag my-image:latest . 
+```
+For more information see the [Using Satellites guide](../cloud/satellites/using.md).
+
+Additionally, all other build options are supported when using `docker-build`. For more information see [build-options](#build-options).
+
+#### Examples
+
+Build a dockerfile in the directory myDockerfiles
+
+```
+earthly docker-build --dockerfile Dockerfile ./myDockerfiles
+```
+
+Push an image built from your Dockerfile built for linux/arm64
+
+```
+earthly docker-build --dockerfile Dockerfile --platform linux/arm64 --tag {DOCKER_TAG} --push ./myDockerfiles
+```
+
+#### Options
+
+##### `--dockerfile <dockerfile-path>`
+
+Specify an alternative Dockerfile to use. The <dockerfile-path> can be either an absolute path on the host system, or relative to <context-path>.
+
+##### `--tag=<image-tag>`
+
+Set the image name and tag to use. This option can be repeated to provide the built image multiple tags.
+
+##### `--target=<target-name>`
+
+Specifies the target to build in a multi-target Dockerfile.
+
+##### `--platform <platform1[,platform2,...]>`
+
+Sets the platform to build for.
+
+{% hint style='info' %}
+##### Note
+Unlike a regular build command, it is possible to specify multiple platforms through this option.
+{% endhint %}
+
+## earthly satellite
 
 #### Synopsis
 
 * ```
-  earthly --version
+  earthly [options] satellite (launch|ls|inspect|select|unselect|rm)
   ```
 
 #### Description
 
-Prints version information about earthly.
+Launch and use a Satellite runner as remote backend for Earthly builds.
 
-## earthly ls
+Satellites can be used to optimize and share cache between multiple builds and users, as well as run builds in native architectures independent of where the Earthly client is invoked.
```

Comment:
optimization seems like a rather strong word to use here, satellites make cache reuse easier -- since cache persists between builds, but there's no optimization process that makes the cache usage the absolute best perfect use. It's just a LRU cache :)

```suggestion
Satellites can be used to share cache between multiple builds and users, as well as run builds in native architectures independent of where the Earthly client is invoked.
```